From: Bram Moolenaar <Bram@vim.org>
Date: Thu, 22 Jun 2017 14:56:12 +0200
Subject: patch 8.0.0651: build failure without the auto command feature

Problem:    Build failure without the auto command feature.
Solution:   Add #ifdef. (closes #1782)

(cherry picked from commit 157069b04e64eff72d2b34a2f727dfb6e2190cf3)

Signed-off-by: James McCoy <jamessan@debian.org>
---
 src/ex_cmds.c | 2 ++
 src/version.c | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/src/ex_cmds.c b/src/ex_cmds.c
index f8893d4..a5b20b5 100644
--- a/src/ex_cmds.c
+++ b/src/ex_cmds.c
@@ -6790,9 +6790,11 @@ fix_help_buffer(void)
     char_u	*rt;
     int		mustfree;
 
+#ifdef FEAT_AUTOCMD
     /* Set filetype to "help" if still needed. */
     if (STRCMP(curbuf->b_p_ft, "help") != 0)
 	set_option_value((char_u *)"ft", 0L, (char_u *)"help", OPT_LOCAL);
+#endif
 
 #ifdef FEAT_SYN_HL
     if (!syntax_present(curwin))
diff --git a/src/version.c b/src/version.c
index a6c4879..36e20ce 100644
--- a/src/version.c
+++ b/src/version.c
@@ -776,6 +776,8 @@ static int included_patches[] =
     706,
 /**/
     703,
+/**/
+    651,
 /**/
     649,
 /**/
